modes.tex

```diff
@@ -156,11 +159,11 @@ \subsection{Exposing Extra C65 Registers}
 \end{verbatim}
 \end{tcolorbox}
 
-Once the program has been entered, type RUN on a new line. This will make the background of the screen fade from
-blue to purple.  If you would like to make the effect progress faster, increase the 0.2 to a larger number, such as 0.5, or to
-make it slower, change it to a smaller number, such as 0.02. You can also change the red component by adding a
-different number to 53504, the green component at 53760 -- 54015 (hex \$D200 -- \$D2FF), or the
-blue component at 54016 -- 54271 (hex \$D300 -- \$D3FF).  For example, to have
+Once the program has been entered, type {\bf RUN} on a new line. This will make the background of the screen fade from
+blue to purple.  If you would like to make the effect progress faster, increase the 0.2 to a larger number such as 0.5. To
+make it slower, change it to a smaller number such as 0.02. You can also change the red component by adding a
+different number to 53504 – 53759 (\$D100 – \$D1FF), the green component at 53760 -- 54015 (\$D200 -- \$D2FF), or the
+blue component at 54016 -- 54271 (\$D300 -- \$D3FF).  For example, to have
```

Comment:
instead of "adding a different number to", can it be "**{\bf POKE}ing** a different number to"

Reply:
fixed. In the context of the sentence `adding` made sense but to keep it all consistent, i changed it to POKEing